crates/trycp_server/src/configure_player.rs

```diff
@@ -57,88 +54,28 @@ pub(crate) fn configure_player(
         players_guard.insert(
             id.clone(),
             Player {
-                admin_port: port,
+                admin_port,
                 processes: Mutex::default(),
             },
         );
     }
 
-    let lair_stderr_log_path = player_dir.join(LAIR_STDERR_LOG_FILENAME);
```

Comment:
Lair is started in-process with holochain now.

Reply:
:guitar: